### ISSUE TITLE
Resulting magic radiodan falls off wifi after 30-60 seconds 

### ISSUE THREAD

**Issue by libbymiller** [radiodan/provision#11]
This might be an issue for Magic button rather than provision. Investigating.

**Comment by libbymiller** [comment]
Reproduced just with

sudo LOG_LEVEL=DEBUG ./provision avahi nginx wpa

Lost connection after about 45 seconds.

```
pi@raspberrypi ~ $ ps ax | grep wpa
1673 ?        Ss     0:00 /sbin/wpa_supplicant -s -B -P /var/run/wpa_supplicant.wlan0.pid -i wlan0 -W -D nl80211,wext -c /etc/wpa_supplicant/wpa_supplicant.conf
2276 ?        Ss     0:00 /sbin/wpa_cli -B -P /var/run/wpa_action.wlan0.pid -i wlan0 -p /var/run/wpa_supplicant -a /sbin/wpa_action
2661 ?        Sl     0:02 /usr/bin/ruby1.9.1 /usr/local/bin/wpa_cli_web start -e production -p 8080
```

**Comment by libbymiller** [comment]
looks like a race condition between wpa_suppplicant and rc.local. Some debugging here:

https://gist.github.com/libbymiller/effe394ba689c2b51e17#file-gistfile1-txt

should be moved to https://github.com/radiodan/wifi-configuration

**Comment by andrewn** [comment]
Fixed by b739851dbaf606 and f1b1cec0594d2e.
Thanks @libbymiller !
